kadai-adapter-camunda-spring-boot-test/src/main/resources/processes/simple_user_task_process_with_different_domains.bpmn

```diff
@@ -73,7 +73,7 @@
     <bpmn:sequenceFlow id="Flow_0jkseco" sourceRef="UserTask_1u5oqhv" targetRef="Gateway_0mqata4" />
     <bpmn:sequenceFlow id="Flow_07abxz0" sourceRef="UserTask_1kfz8x" targetRef="Gateway_0mqata4" />
     <bpmn:sequenceFlow id="Flow_0h2nc8t" sourceRef="Gateway_0mqata4" targetRef="EndEvent_0jgafnj" />
-    <bpmn:serviceTask id="Task_0sh7pjm" name="ComplexVariableSetter" camunda:class="pro.taskana.impl.ComplexProcessVariableSetter">
+    <bpmn:serviceTask id="Task_0sh7pjm" name="ComplexVariableSetter" camunda:class="impl.io.kadai.ComplexProcessVariableSetter">
```

Comment:
io.kadai.impl.ComplexProcessVariableSetter ?

Reply:
You're right, I will change it